pkg/apis/core/types_shoot.go

```diff
@@ -307,6 +307,7 @@ type ETCDEncryptionKeyRotation struct {
 	LastInitiationTime *metav1.Time
 	// LastInitiationFinishedTime is the recent time when the ETCD encryption key credential rotation initiation was
 	// completed.
+	// Deprecated: This field will be removed in gardener `v1.130`. The field is not longer needed with https://github.com/gardener/gardener/pull/12442.
```

Comment:
Cosmetic suggestion to separate docstring from deprecation notice.
```suggestion
        // 
	// Deprecated: This field will be removed in gardener `v1.130`. The field is not longer needed with https://github.com/gardener/gardener/pull/12442.
```

Reply:
Btw why did you decide to remove it in `v1.130`? If some users had automation that depended on this field perhaps it should be coupled to a k8s version: https://github.com/gardener/gardener/blob/7a88612433828dda9ec97409b6edf2e70d9d89a4/docs/development/process.md?plain=1#L207-L209

/cc @shafeeqes since the deprecation idea came in https://github.com/gardener/gardener/pull/12442#discussion_r2191768504

Reply:
Could you please also add a TODO statement specifying when the field will be removed and who is responsible for it, eg:  
```suggestion
	// Deprecated: This field will be removed in gardener `v1.130`. The field is not longer needed with https://github.com/gardener/gardener/pull/12442.
	// TODO(AleksandarSavchev): Remove this field after Gardener `v1.129` has been released.
```

Reply:
Typo, and could you add a human readable description instead of link to the PR? E.g.:
```suggestion
	// Deprecated: This field will be removed in gardener `v1.130`. The field is no longer needed because the `completing` phase of the etcd encryption is triggered automatically after the `preparing` phase finishes.
```

Reply:
> Btw why did you decide to remove it in v1.130? If some users had automation that depended on this field perhaps it should be coupled to a k8s version

Version `v1.130` was chosen as far enough in the future (~2 months) for all `TODO: remove`s in this PR.